### ISSUE TITLE
[Screen Reader-Make Code - Import]: There is no heading level defined on the 'Clone or create your own GitHub repo' pop up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3753]
**User Experience:** 
H1 is mainly defined or related to the page title, and if there is no H1 tag is defined then AT user who depends upon screen reader get disoriented or not able to understand the main purpose of the opened page.

**Note:** User credentials should NOT be included in the bug.

**Repro Steps:**

1. Launch application URL
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to "Your GitHub Repo" option and  Select it.
6. Navigate and select "Sign In" control under Sign in with GitHub and navigate to all the controls of dialogue "Clone or create your own GitHub repo
7. Verify the issue.

**Actual Result:**
There is no  heading are defined on the 'Clone or create your own GitHub repo' pop up.

**Expected Result:**
Heading should be uniquely defined on the 'Clone or create your own GitHub repo' pop up.

**Note:**
Same issue appear under Chrome & Firefox.
Same issue exists throughout the application for all type of pop ups.

**WCAG Reference:**
[MAS 2.4.6 – Headings and Labels](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #3725 ; #Win10-Edge(Chromium)

![Uploading WCAG2.4.6.png…]()
